packages/api/src/resolvers/types/Service.ts

```diff
@@ -2,8 +2,8 @@
  * Copyright (c) Microsoft. All rights reserved.
  * Licensed under the MIT license. See LICENSE file in the project.
  */
-import { Tag, Service as ServiceType, ServiceResolvers } from '@cbosuite/schema/dist/provider-types'
-import { createGQLTag } from '~dto'
+import { Tag, Service as ServiceType, ServiceResolvers } from '@cbosuite/schema/lib/provider-types'
```

Comment:
you want dist/provider_types here

Reply:
thanks. I have applied the change to the impacted files